src/styles/index.ts

```diff
@@ -455,6 +455,10 @@ const styles = (theme: ThemeColors) =>
             fontWeight: FontUtils.fontWeight.bold,
         },
 
+        textLineHeightXLarge: {
```

Comment:
This feels like a very misleading name for this... I wouldn't consider this line height to be XLarge, I would consider it to be our default line height for our default font size.

Reply:
Does it mean updating the `baseFontStyle` or creating a more suitable name?

Reply:
Can you confirm that our base font style includes line height? And if so, what is the line height? Because maybe we don't actually need to do anything here and we can just use our "default" text styles.